javascript/autoFillSearchParams.js

```diff
@@ -0,0 +1,45 @@
+class AutoFillSearchParams {
+    searchParams = new URLSearchParams(decodeURIComponent(location.search));
+    result = '';
+    paramsKeys = ['prompt', 'negative_prompt', 'size', 'seed', 'steps', 'sampler', 'cfg', 'clip_skip', 'batch_size', 'ensd']
+
+    joinParams(key, value) {
+        const joinValue = `${key.replace('_', ' ').replace(key[0],key[0].toUpperCase())}:${value}`;
+        // set to the front
+        if (key === 'prompt') {
+            this.result = `${value}__promot__\r\n${this.result}`;
+            return;
+        }
+        if (key === 'negative_prompt') {
+            this.result = this.result.replace('__promot__', `\r\n${joinValue}`);
```

Comment:
It's possible that `negative_prompt` was read before `prompt`, and line 14 will fail.